### PR TITLE
Fix DataProviders links

### PR DESCRIPTION
**Pull request by krismas**

### CODE REVIEW

File: core/data-providers.md [hunk 1]
```diff
@@ -64,12 +64,11 @@ services:
 
 Tagging the service with the tag `api_platform.collection_data_provider` will enable API Platform Core to automatically
 register and use this data provider. The optional attribute `priority` allows to define the order in wich are called the
-data providers. The first data provider not throwing a `ApiPlatform\Core\Exception\ResourceClassNotSupportedException` will
-be used.
+data providers. The first data provider not throwing a `ApiPlatform\Core\Exception\ResourceClassNotSupportedException` will be used.
```
Comment (teohhanhui) [inline core/data-providers.md]: Please revert this. The soft line length limit is at 120 characters, it'd seem.

**Comment by krismas** [comment]
I will do that... but in the documentation many lines exceeds this limit ! What are the rules?

**Comment by teohhanhui** [comment]
> but in the documentation many lines exceeds this limit ! What are the rules?

@api-platform/core-team?

**Comment by theofidry** [comment]
If would say 120 by default but I don't really think it's worth to enforce strict rules.

**Comment by dunglas** [comment]
Agree with @theofidry. For now I use the following rule: insert a LF char instead of the next blank character after the 120th column.

**Comment by dunglas** [comment]
Thanks @krismas

**Pull request merged**